### PR TITLE
chore(sites): sites_entity subtable attributes are moved to metadata

### PR DESCRIPTION
**Pull request by jdalsem**
Ref #2675

BREAKING CHANGE:
The sites_entity table no longer exists. Update your queries if needed.

### CODE REVIEW

**Comment by jdalsem** [comment]
Removed some obsolete test files for ElggSite base attribute checks as they are the same as ElggEntity

File: docs/admin/duplicate-installation.rst [hunk 1]
```diff
@@ -138,6 +138,8 @@ We need to change these lines to match our new installation:
    // If you're sharing a database with other applications, you will want to use this
    // to differentiate Elgg's tables.
    $CONFIG->dbprefix = 'elgg';
+   
+   $CONFIG->wwwroot = 'http://your.website.com'
```
Comment (jeabakker) [inline docs/admin/duplicate-installation.rst]: add trailing `/`

File: engine/classes/ElggEntity.php [hunk 2]
```diff
@@ -1395,39 +1395,40 @@ protected function create() {
...
 			$params = [
 				':guid' => $guid,
 			];
-			$db->deleteData($query, $params);
-
-			_elgg_services()->entityCache->remove($guid);
-
-			throw new \IOException("Unable to save new object's secondary entity information!");
+			foreach ($attrs as $attr) {
+				$params[":$attr"] = ($attr === 'url') ? '' : (string) $this->attributes[$attr];
```
Comment (jeabakker) [inline engine/classes/ElggEntity.php]: is this `if` still needed as `url` is no longer an attribute

File: engine/classes/ElggEntity.php [hunk 3]
```diff
@@ -1509,28 +1510,28 @@ protected function update() {
...
-		if ($db->updateData($query, false, $params) === false) {
-			return false;
+		if (!empty($attrs)) {
+			$sets = array_map(function ($attr) {
+				return "$attr = :$attr";
+			}, $attrs);
+			$sets = implode(', ', $sets);
+	
+			foreach ($attrs as $attr) {
+				$params[":$attr"] = ($attr === 'url') ? '' : (string) $this->attributes[$attr];
```
Comment (jeabakker) [inline engine/classes/ElggEntity.php]: is this `if` still needed as `url` is no longer an attribute

File: engine/classes/ElggEntity.php [hunk 4]
```diff
@@ -1594,7 +1595,8 @@ private function getSecondaryTableColumns() {
 			return ['name', 'description'];
 		}
 		if ($this instanceof ElggSite) {
-			return ['name', 'description', 'url'];
+// 			return ['name', 'description', 'url'];
```
Comment (jeabakker) [inline engine/classes/ElggEntity.php]: don't leave unused code behind

File: engine/classes/ElggEntity.php [hunk 5]
```diff
@@ -1634,9 +1636,9 @@ public static function getExtraAttributeDefaults($type) {
 				];
 			case 'site':
 				return [
-					'name' => null,
-					'description' => null,
-					'url' => null,
+// 					'name' => null,
+// 					'description' => null,
+// 					'url' => null,
```
Comment (jeabakker) [inline engine/classes/ElggEntity.php]: don't leave unused code behind

**Review by jeabakker — CHANGES_REQUESTED** [review]

**Comment by jdalsem** [comment]
addressed all comments

**Review by jeabakker — APPROVED** [review]

**Pull request merged**